types/amcharts/index.d.ts

```diff
@@ -1,6 +1,7 @@
 // Type definitions for amCharts
```

Comment:
Consider create a file named `amcharts-tests.ts`

Then `import` and code related to this PR changes.

Reply:
To start the tests, I provided only a small one.
First I could imagine removing the last disabled rules in tslint.json - but in a new PR - after this one is merged?

This PR seems more valuable to me, having strict mode fixed and afterwards providing linted files + tests ;-)

Reply:
> First I could imagine removing the last disabled rules in tslint.json - but in a new PR - after this one is merged?
Yes, that would be better. ( ^^)b